modules/n1ql/pages/n1ql-language-reference/using-ai.adoc

```diff
@@ -699,8 +704,20 @@ List the names and cities of hotels with a rating greater than 4;
 ----
 ====
 
+[[using-ai-limitations]]
+== Limitations 
+
+When using the statement, consider the following:
+
+* To run the statement on Capella, use either the cbq shell or an SDK.
+The statement does not function as expected when executed directly through the Capella UI (Query Workbench).
+* For similar functionality within the Capella UI, use xref:get-started:capella-iq/work-with-capellaiq.adoc[Capella iQ] instead.
+* You cannot use the `creds` and `natural_cred` parameters with the following types of credentials:
+** Single Sign-On (SSO)
```

Comment:
Can you please check whether I should add a note about SSO, MFA, and social login to the Data API reference as part of DOC-13662?